zebra/zebra_nhg.c

```diff
@@ -688,18 +689,26 @@ static void handle_recursive_depend(struct nhg_connected_tree_head *nhg_depends,
 		depends_add(nhg_depends, depend);
 }
 
+static bool zebra_need_to_create_pic(struct nexthop *nh)
+{
+	if (nh && nh->nh_srv6 && !sid_zero(nh->nh_srv6->seg6_segs))
```

Comment:
It should be better if the pic configuration was configurable.
by the way; does it make sense to apply this only for SRv6 ?
what about MPLS ? and what about regular IP routes?

Reply:
Yes. Previously, we only used the PIC design for the SRv6 scenario. In fact, the PIC design is also used for MPLS and VXLAN scenarios. I will modify this logic to be more generic later. Thank you.